spec/aws-record/record/dirty_tracking_spec.rb

```diff
@@ -236,6 +260,39 @@
 
   end
 
+  describe "#update" do
```

Comment:
It also looks like we never actually test-call the `#update` method in these tests.